mkdocs.yml

```diff
@@ -1,95 +1,91 @@
 site_name: OpenSAFELY documentation
 nav:
-    - Introduction: index.md
-    - How OpenSAFELY works:
-        - Our philosophy: open-methods.md
-        - Security: security-levels.md
-        - Analysis workflow: workflow.md
+  - Introduction: index.md
+  - How OpenSAFELY works:
+      - Our philosophy: open-methods.md
+      - Security: security-levels.md
+      - Analysis workflow: workflow.md
+  - Getting Started guide: getting-started.md
+  - Installation guides:
```

Comment:
Changing to `Installation Reference Guide` maybe - otherwise its repetitive if done tutorial